apps/testing-javascript/.env.production

```diff
@@ -5,8 +5,7 @@ NEXT_PUBLIC_APP_NAME=testing-javascript
 INNGEST_APP_NAME=testing-javascript
 NEXT_PUBLIC_SITE_TITLE="Testing JavaScript"
 NEXT_PUBLIC_HOST=testingjavascript.com
-# NEXT_PUBLIC_URL=https://testingjavascript.com
-NEXT_PUBLIC_URL=https://testing-javascript.vercel.app
+NEXT_PUBLIC_URL=https://testingjavascript.com
```

Comment:
@jbranchaud I guess it should be fixed in this way?

Reply:
I believe it is overridden by vercel env vars, but still a good thing to update. 🙌

Reply:
not sure, there is no such env at TJS vercel:
![CleanShot 2024-01-10 at 16 19 36@2x](https://github.com/skillrecordings/products/assets/1519448/738a0ef6-020e-43f6-bf13-cfc49e824bb9)